cps/src/main/js/workflow-editor.js

```diff
@@ -48,7 +48,34 @@ jenkinsJSModules.import('ace-editor:ace-editor-122')
                 });
 
                 editor.on('blur', function() {
-                    textarea.trigger('change');
```

Comment:
> I do not get why you cannot simply restore the call to trigger here

I could, but then two AJAX calls will be done on textarea `blur`, one to `doCheckScript` and a second one to `doCheckScriptJson` which is a kind of duplicated work to validate the script. Moreover, the ugly error message under the textarea will be shown (which is redundant since we have the error in-lined in the editor).

I think the best option is to remove the last commit and show both syntax errors and approval requirements as ACE editor annotations. We could even remove the old `doCheckScript`. Do you agree?